addons/l10n_it_edi/models/account_move.py

```diff
@@ -323,7 +323,7 @@ def _l10n_it_edi_add_base_lines_xml_values(self, base_lines_aggregated_values, i
 
             # Price unit.
             if quantity:
-                it_values['prezzo_unitario'] = base_line['gross_price_subtotal'] / quantity
+                it_values['prezzo_unitario'] = base_line['currency_id']._convert(base_line['gross_price_subtotal'] / quantity, self.company_currency_id, date=self.date)
```

Comment:
I think we should only call `_convert()` when necessary, to avoid the rounding:
https://github.com/odoo/odoo/blob/76e2aa6b70d1fb03b8ce2f8e4a7e9c3d9425270f/odoo/addons/base/models/res_currency.py#L291-L292